packages/widgetbook/lib/src/fields/duration_field.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+class DurationField extends Field<Duration> {
+  DurationField({
+    required super.name,
+    required Duration super.initialValue,
+    super.onChanged,
+  }) : super(
+          type: FieldType.doubleInput,
+          codec: durationCodec,
+        );
+
+  @override
+  Widget toWidget(BuildContext context, String label, Duration? currentValue) {
+    return TextFormField(
+      initialValue:
+          codec.toParam(currentValue ?? initialValue ?? Duration.zero),
+      keyboardType: TextInputType.number,
```

Comment:
We can also add a `suffix` of "ms" to make sure this field is visually different from other fields. This can also help users know the unit we are using, leading into a better UX.

What do you think?

```suggestion
      keyboardType: TextInputType.number,
      decoration: const InputDecoration(
        suffix: Text('ms'),
      ),
```

Reply:
I agree. User would be able to tell the unit. Leaves no room for confusion.